sdk/core/Azure.Core.Expressions.DataFactory/src/DataFactoryElementJsonConverter.cs

```diff
@@ -67,78 +67,6 @@ public override bool CanConvert(Type typeToConvert)
             throw new InvalidOperationException($"Unable to convert {typeToConvert.Name} into a DataFactoryElement<T>");
         }
 
-        public override void Write(Utf8JsonWriter writer, object? value, JsonSerializerOptions options)
```

Comment:
We should just redirect to the MRW implementation. We can't break existing usages of JsonSerializer.Serialize(dataFactoryElement), as this is a public type.

Reply:
Explained in above comment

Reply:
added it back